packages/gatsby-plugin-mdx/loaders/mdx-loader.js

```diff
@@ -3,6 +3,7 @@ const { getOptions } = require(`loader-utils`)
 const grayMatter = require(`gray-matter`)
 const unified = require(`unified`)
 const babel = require(`@babel/core`)
+const createRequireFromPath = require(`gatsby/dist/utils/create-require-from-path`)
```

Comment:
We should not import internals like that as they are not considered public API and can change at any time.

If there are utils from `gatsby` core package that other packages want to use, let's move that util to https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-core-utils and import this both in `gatsby` and in here

Reply:
(the move can be in the same PR)

Reply:
Sure, will do!